ilan/install_ihs

```diff
@@ -19,7 +19,9 @@
 VERSION=$1
 IBM_ID=$2
 IBM_PASSWORD=$3
-
+JAVA_NEEDED="no"
+JAVA_PACKAGE=""
+echo "Running build for versions: "$VERSION
```

Comment:
I'd put this echo in "build_all". If you're building only a single image you wouldn't need this.